src/vmm/src/version_map.rs

```diff
@@ -17,9 +17,9 @@ lazy_static! {
 
     /// Static instance used for mapping Firecracker release version to
     /// snapshot data format version.
-    pub static ref FC_VERSION_TO_SNAP_VERSION: HashMap<u16, u16> = {
+    pub static ref FC_VERSION_TO_SNAP_VERSION: HashMap<String, u16> = {
```

Comment:
The design is to have 1:1 mapping between the string and the u16. I think it is worth stating this here. We still need to figure out  a way to avoid manually inserting a line here for each release.